### PR TITLE
docs: CI testing intro

### PR DESCRIPTION
**Pull request by nissimsan**
Intro to interop testing

***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/traceability-interop/pull/231.html" title="Last updated on May 23, 2022, 9:55 AM UTC (6c49d29)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/traceability-interop/231/6a854bc...6c49d29.html" title="Last updated on May 23, 2022, 9:55 AM UTC (6c49d29)">Diff</a>

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -307,6 +307,35 @@ <h2>Terminology</h2>
...
+    <section class="informative">
+      <h2>Interoperability Testing</h2>
+      <p>
+        Participating platforms are proving actual interoperability by enroling in continuous integration tests. These are based on postman 
+        collections which orchestrate the interaction of participating parties, for example `Credential Issuance`, `Credential Verification`, 
+        `Presentation Exchange`, etc. 
+        These postman collections are described with tutorials here: https://github.com/w3c-ccg/traceability-interop/tree/main/docs/tutorials
+      </p>
+      <p>
+        An interoperability report is continuously created from the results of executing these collections, targeted the participating
```
Comment (brownoxford) [inline docs/index.html:319]:
```suggestion
        An interoperability report is continuously created from the results of executing these collections, targeting the participating 
```

File: docs/index.html [hunk 2]
```diff
@@ -307,6 +307,35 @@ <h2>Terminology</h2>
 
     <section id="conformance"></section>
 
+    <section class="informative">
+      <h2>Interoperability Testing</h2>
+      <p>
+        Participating platforms are proving actual interoperability by enroling in continuous integration tests. These are based on postman 
+        collections which orchestrate the interaction of participating parties, for example `Credential Issuance`, `Credential Verification`,
```
Comment (TallTed) [inline docs/index.html:314]:
```suggestion
        collections which orchestrate the interaction of participating parties; for example, `Credential Issuance`, `Credential Verification`, 
```

File: docs/index.html [hunk 3]
```diff
@@ -307,6 +307,35 @@ <h2>Terminology</h2>
...
+        Participating platforms are proving actual interoperability by enroling in continuous integration tests. These are based on postman 
+        collections which orchestrate the interaction of participating parties, for example `Credential Issuance`, `Credential Verification`, 
+        `Presentation Exchange`, etc. 
+        These postman collections are described with tutorials here: https://github.com/w3c-ccg/traceability-interop/tree/main/docs/tutorials
+      </p>
+      <p>
+        An interoperability report is continuously created from the results of executing these collections, targeted the participating 
+        platforms. In cases of multi-party interactions (such as party A making a verifiable presentation to party B), all combinations of 
+        platform interchange are executed. 
+        The execution of the continuous interoperability report is done with Github Actions on the Traceability Interoperability github
```
Comment (TallTed) [inline docs/index.html:322]:
```suggestion
        The execution of the continuous interoperability report is done with GitHub Actions on the Traceability Interoperability GitHub 
```

**Review by OR13 — APPROVED** [review]
with suggestions applied.

File: docs/index.html [hunk 4]
```diff
@@ -307,6 +307,35 @@ <h2>Terminology</h2>
...
+        platform interchange are executed. 
+        The execution of the continuous interoperability report is done with GitHub Actions on the Traceability Interoperability GitHub 
+        repository: https://github.com/w3c-ccg/traceability-interop/actions/workflows/interoperability-report.yml
+      </p>
+      <p>
+        In each test, the GitHub Action runner carries out a number of invocations, acting on behalf of varying parties taking different roles 
+        (`issuer`, `holder` and `verifier`), in turn invoking the participating platform. 
+        Below is an example of calls made by the runner as it is executing a DIDAuth Presentation Exchange flow:
+        <ul>
+          <li>Runner acts as party A in the role of `issuer`, calling `/credentials/issue` on platform X
```
Comment (brownoxford) [inline docs/index.html:330]: Is platform X not acting as the issuer here?

File: docs/index.html [hunk 5]
```diff
@@ -307,6 +307,35 @@ <h2>Terminology</h2>
 
     <section id="conformance"></section>
 
+    <section class="informative">
+      <h2>Interoperability Testing</h2>
+      <p>
+        Participating platforms are proving actual interoperability by enrolling in continuous integration tests. These are based on Postman 
+        collections which orchestrate the interaction of participating parties; for example, `Credential Issuance`, `Credential Verification`, 
+        `Presentation Exchange`, etc.
```
Comment (mkhraisha) [inline docs/index.html:315]:
```suggestion
        and `Presentation Exchange`.
```

File: docs/index.html [hunk 6]
```diff
@@ -307,6 +307,35 @@ <h2>Terminology</h2>
 
     <section id="conformance"></section>
 
+    <section class="informative">
+      <h2>Interoperability Testing</h2>
+      <p>
+        Participating platforms are proving actual interoperability by enrolling in continuous integration tests. These are based on Postman
```
Comment (mkhraisha) [inline docs/index.html:313]:
```suggestion
        Participating platforms are proving actual interoperability by enrolling in continuous integration tests. These tests are based on Postman 
```

File: docs/index.html [hunk 7]
```diff
@@ -307,6 +307,35 @@ <h2>Terminology</h2>
...
+        Participating platforms are proving actual interoperability by enrolling in continuous integration tests. These are based on Postman 
+        collections which orchestrate the interaction of participating parties; for example, `Credential Issuance`, `Credential Verification`, 
+        `Presentation Exchange`, etc. 
+        These Postman collections are described with tutorials here: https://github.com/w3c-ccg/traceability-interop/tree/main/docs/tutorials
+      </p>
+      <p>
+        An interoperability report is continuously created from the results of executing these collections, targeting the participating 
+        platforms. In cases of multi-party interactions (such as party A making a verifiable presentation to party B), all combinations of 
+        platform interchange are executed. 
+        The execution of the continuous interoperability report is done with GitHub Actions on the Traceability Interoperability GitHub
```
Comment (mkhraisha) [inline docs/index.html:322]:
```suggestion
        The Interoperability report is executed by a GitHub Action on the Traceability Interoperability GitHub 
```

File: docs/index.html (hunk 4 above)
Comment (mkhraisha) [inline docs/index.html:330]:
```suggestion
          <li>Runner calls `/credentials/issue` on Party A's platform, allowing Party A to act as the issuer.
```

Comment (mkhraisha) [reply docs/index.html:330]: I think similar changes across the board going down the list

**Review by mkhraisha — CHANGES_REQUESTED** [review]
Some suggestions to help with clarity

File: docs/index.html (hunk 4 above)
Comment (nissimsan) [reply docs/index.html:330]: I am trying to distinguish between the platforms and the actors - easier said than done. :) I'll give the wording another iteration.

Comment (nissimsan) [reply docs/index.html:330]: I like this, @mkhraisha. "Party A's platform" reads much clearer than "Platform X".

**Comment by nissimsan** [comment]
@brownoxford  and @mkhraisha, I've reworded the list of runner acting in different roles. Especially your suggestion, Mahmoud, "Party A's platform" reads much better than "Platform X". I find it much better now - thanks!

**Review by brownoxford — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**